lib/spdy-transport/protocol/http2/parser.js

```diff
@@ -386,6 +417,28 @@ Parser.prototype.onSettingsFrame = function onSettingsFrame(header,
       settings[name] = value;
   }
 
+  if (settings['enable_push'] !== undefined) {
+    if ((settings['enable_push'] !== 0)
+     && (settings['enable_push'] !== 1)) {
+      return callback(this.error(constants.error.PROTOCOL_ERROR,
+                                 'SETTINGS_ENABLE_PUSH must be 0 or 1'));
+     }
+  }
+  if (settings['initial_window_size'] !== undefined) {
+    if ((settings['initial_window_size'] > constants.MAX_INITIAL_WINDOW_SIZE)
+     || (settings['initial_window_size'] < 0)) {
+      return callback(this.error(constants.error.FLOW_CONTROL_ERROR,
+                                 'SETTINGS_INITIAL_WINDOW_SIZE is OOB'));
+     }
+  }
+  if (settings['max_frame_size']) {
+    if ((settings['max_frame_size'] > constants.ABSOLUTE_MAX_FRAME_SIZE)
+     || (settings['max_frame_size'] < constants.INITIAL_MAX_FRAME_SIZE)) {
```

Comment:
Does it pass `npm test`? There is a linting phase running after all tests, and it should catch most of the linting errors, including "operator must be at the end of the line".